pkgs/top-level/all-packages.nix

```diff
@@ -34245,4 +34245,6 @@ with pkgs;
   };
 
   zthrottle = callPackage ../tools/misc/zthrottle { };
+
```

Comment:
Why did you put this at the end of the file? It makes more sense to put under the `### DEVELOPMENT/INTERPRETERS` on the file.